lib/routes/attachment.js

```diff
@@ -108,11 +108,11 @@ module.exports = function(crowi, app) {
           // TODO size
           return Attachment.create(id, req.user, filePath, originalName, fileName, fileType, fileSize);
         }).then(function(data) {
-          var imageUrl = fileUploader.generateUrl(data.filePath);
+          var attachment = data.toObject();
+          attachment["url"] = fileUploader.generateUrl(data.filePath);
```

Comment:
attachment should be the object of attachment model imo,
just replace `result.filename` to `result.url` ?